### PR TITLE
Modified my file

### PR DESCRIPTION
**Pull request by cityofmiami**

### CODE REVIEW

**Comment by cityofmiami** [comment]
Added a caption to my slide. Ticket 23AFR.

File: _slides/05-cityofmiami.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+City of Miami GitHub
```
Comment (beardofedu) [inline _slides/05-cityofmiami.md:6]: This is great! _But_ you didn't reference your image 😭

File: _slides/05-cityofmiami.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
 ![Robotocat](https://octodex.github.com/images/Robotocat.png)
```
Comment (beardofedu) [inline _slides/05-cityofmiami.md:8]: If you don't like this image, you can grab a new one @ https://octodex.github.com

File: _slides/05-cityofmiami.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (beardofedu) [inline _slides/05-cityofmiami.md:3]:
```suggestion
title: "Welcome to cityofmiami's slide!"
```

**Review by beardofedu — CHANGES_REQUESTED** [review]
This looks great , 🚢 🚢 🚢 🚢 🚢 🚢 🚢

**Review by beardofedu — APPROVED** [review]

**Pull request merged**